tsp-output/@azure-tools/typespec-csharp/src/Generated/Internal/SerializedModelData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenAI;
+
+internal class SerializedModelData : BinaryData
```

Comment:
For now, we can emit this as an internal type -- the same client will deserialize and serialize.  We can later move this to a MRW API.